azerothcore/azerothcore-wotlk
# fix(Core/Instances): Always put boss minions in combat with zone

## Changes Proposed:
-  Always put minions in combat with all players, even if they already have a target.
-  Previously this call was void if they were already attacking something, causing them to not be put in combat with every player, eventually leading them to reset if their victim died before another player engaged them.

## Issues Addressed:

- Closes 

## SOURCE:


## Tests Performed:

- Not tested.
- 


## How to Test the Changes:


1. Engage Gehennas with two characters, attack the boss with both characters but only hit the add with the tank
2. Kill the tank that pulled
3. Notice if the adds stay in combat or reset without attacking the other player

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
